### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MajesticBeast**
Modified and simplified Self-Hosting -- Docker section

### CODE REVIEW

**Comment by MajesticBeast** [comment]
I think the direction the README should take is to give a quick "get up and running" method. This is the first step in that direction by removing all the unneeded Redis info and confusing/duplicate/poorly formatted information for Docker self-hosting. This new docker-compose code and curl code is essentially a copy/paste of what I'm doing with my setup. It will eventually be expanded with extra options/variables/redis info.

This is also literally my first pull request on Github. Hopefully I'm doing this right.

File: README.md [hunk 1]
```diff
@@ -202,47 +202,28 @@ If you are interested please see the [contact info on my github page](https://gi
...
-      - DB=/dst.db
-      - CACHE=true
-    volumes:
-      - /home/infra/dst.db:/dst.db
-    ports:
-      - "8112:8080"
+ discordstockticker:
+  image: ghcr.io/rssnyder/discord-stock-ticker:3.9.3
+  environment:
+    -CACHE=false  # Set to true if your bot:coin ratio is going to be greater than 1
```
Comment (rssnyder) [inline README.md:218]:
```suggestion
```
lets remove this, almost no one should ever need a cache

File: README.md [hunk 2]
```diff
@@ -202,47 +202,28 @@ If you are interested please see the [contact info on my github page](https://gi
...
+    -FREQUENCY=30 # How often to update all tickers, in seconds
+    -LOGLEVEL=0   # 0=production builds. 1=dev builds. Set to 1 if you are debugging/troubleshooting
+  ports:
+   - "8080:8080"  # Port you want to use for your API
```
Comment (rssnyder) [inline README.md:222]:
```suggestion
   - "8080:8080"  # The first number is the port you want to use for your API
```

File: README.md [hunk 3]
```diff
@@ -202,47 +202,28 @@ If you are interested please see the [contact info on my github page](https://gi
...
 ```
 
+These instructions do not take advantage of Redis to store your tickers for if/when you stop your Docker container. Those instructions are yet to come. In the meantime, you can create a shell script (assuming you're running Linux) to start all your bots at once.
```
Comment (rssnyder) [inline README.md:225]:
```suggestion
```
unnessesary

File: README.md [hunk 4]
```diff
@@ -300,23 +281,44 @@ Now that you have the service running, you can add bots using the API exposed on
...
 
-If you are new to use an API to manage things, there are several ways to make the calls:
+If you are new to using an API to manage things, there are several ways to make API calls:
 
-curl:
+1) Curl. This is a command available on virtually all Linux distros. Replace anything between < and > with the appropriate information.
+
+The generic format for a curl API call:
+```shell
+curl -X <method> -H "Content-type: application/json" -d <inline json or from file> hostname:port/<bot type>
```
Comment (rssnyder) [inline README.md:301]:
```suggestion
curl -X <method> -H "Content-type: application/json" -d <inline json or from file> <hostname>:<port>/<bot type>
```

File: README.md (hunk 1 above)
Comment (MajesticBeast) [reply README.md:218]: Probably best. Just wasn't sure if you wanted to keep that in there or not.

File: README.md (hunk 3 above)
Comment (MajesticBeast) [reply README.md:224]: I may be not understanding something here then. If I docker-compose down, then bring it back up. My bots are gone.

Comment (rssnyder) [reply README.md:224]: Redis has nothing to do with storing the state of you bots, Redis is for the cache system.

What you can do is use the DB option and docker volumes to store state in a local db file that gets reloaded on every restart. I will make a suggestion to add it.

File: README.md [hunk 5]
```diff
@@ -202,47 +202,27 @@ If you are interested please see the [contact info on my github page](https://gi
...
-      - /home/infra/dst.db:/dst.db
-    ports:
-      - "8112:8080"
+ discordstockticker:
+  image: ghcr.io/rssnyder/discord-stock-ticker:3.9.3
+  environment:
+    -FREQUENCY=30 # How often to update all tickers, in seconds
+    -LOGLEVEL=0   # 0=production builds. 1=dev builds. Set to 1 if you are debugging/troubleshooting
+  ports:
+   - "8080:8080"  # The first number is the port you want to use for your API
```
Comment (rssnyder) [inline README.md:221]:
```suggestion
 discordstockticker:
  image: ghcr.io/rssnyder/discord-stock-ticker:3.9.3
  environment:
    - FREQUENCY=30 # How often to update all tickers, in seconds
    - LOGLEVEL=0       # 0=production builds. 1=dev builds. Set to 1 if you are debugging/troubleshooting
    - DB=/state/bots.db # where to store your bots (a non existing file within the container) to have them persist on restarts
    volumes:
      - <local path to folder to store db in>:/state
  ports:
   - "8080:8080"  # The first number is the port you want to use for your API
```
This adds docs on how to store your running bots in a local database file, that lets users restart the container anytime and have the bots restart with them.

**Review by rssnyder — APPROVED** [review]
TYSM

**Pull request merged**